python/tox.ini

```diff
@@ -39,7 +43,7 @@ omit =
   *.jinja2
 
 [report]
-fail_under = 90.00
+fail_under = 80.00
```

Comment:
It was originally 80, and I did adjust it down at one point because I was doing some code gen that made it super hard to meet that number without a ton of test code generation being added as well.  In the end, I change the code gen approach, resolved the issue, and was able to put the number back to 80 (and pass it).